Tasks/MavenAuthenticateV0/task.json

```diff
@@ -0,0 +1,68 @@
+{
+    "id": "D4B964F9-EA90-41BB-9526-29589628AD90",
+    "name": "MavenAuthenticate",
+    "friendlyName": "Maven Authenticate",
+    "description": "Provides credentials for Azure Artifacts feeds and service connections for maven repositories",
+    "helpUrl": "https://go.microsoft.com/fwlink/?linkid=2099968",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?linkid=2099968)",
+    "category": "Package",
+    "author": "Microsoft Corporation",
+    "version": {
+        "Major": 0,
+        "Minor": 1,
+        "Patch": 0
+    },
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
+    ],
+    "minimumAgentVersion": "2.115.0",
+    "instanceNameFormat": "Maven Authenticate",
+    "inputs": [
+        {
+            "name": "feeds",
+            "type": "pickList",
+            "label": "Feeds",
+            "defaultValue": "",
+            "helpMarkdown": "Select one or multiple feeds to authenticate.",
+            "properties": {
+                "EditableOptions": "true",
+                "MultiSelectFlatList": "true"
+            }
+        },
+        {
+            "name": "serviceEndpoints",
```

Comment:
Same comment as above. Check the spec to make sure that the yaml key name is correct.

Reply:
Fixed